### ISSUE TITLE
谷歌盘里面的权重文件是准确性72.0% 的版本么，还是说需要我们自行重新训练啊

### ISSUE THREAD

**Issue by aslily1234** [cardwing/Codes-for-Lane-Detection#119]
1、首先感谢开源了一个车道线检测的新模型~
2、想跟您确认一下谷歌盘里面的权重文件是基于哪个训练集训练得到的，直接跟SCNN那个模型做对比，感觉效果并没有之前那个好。

**Comment by cardwing** [comment]
The trained model, i.e., ENet-label-new.t7, is trained on CULane training set and achieves 72.0 on CULane testing set. It outperforms SCNN (71.6) by 0.4 points. Using the training script you can also train the model which finally achieves around 72.0.
